common/collation/root.xml

```diff
@@ -947,20 +947,20 @@ For terms of use, see http://www.unicode.org/copyright.html
 &龠=\uFDD0龠 # radical 214
 				]]></cr>
 			</collation>
-<!-- Machine-readable version of the emoji ordering rules for v12.0 (corresponding to CLDR). -->
+<!-- Machine-readable version of the emoji ordering rules for v13.0 (corresponding to CLDR). -->
 <collation type='emoji'>
 <cr><![CDATA[
 # START AUTOGENERATED EMOJI ORDER
 & [last primary ignorable]<<*🦰🦱🦳🦲🏻🏼🏽🏾🏿
 & [before 1]﷑€
```

Comment:
Is that a U+FFFD before the € sign? Does that work with the ICU Collator?

Reply:
It is U+FDD1 and is already present in the ICU maint/maint-66 collation data, so no change there.

Reply:
Oh, that makes sense.
Sorry, I just looked at it with a browser extension for Unicode properties, and I guess that turns noncharacter code points into U+FFFD for some reason...
Should have looked at the actual file.

Reply:
PS: It would be clearer to have that one be escaped in the collation rules, as \uFDD1

Reply:
Good idea.

(I want to get unicodetools moved to github after the release; then we can keep track of improvements like this!)